pages/common/aws-ec2.md

```diff
@@ -0,0 +1,36 @@
+# aws ec2
+
+> CLI for AWS EC2  - provides secure and resizable computing capacity in the AWS cloud to enable faster development and deployment of applications.
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/ec2/index.html#cli-aws-ec2i>.
+
+- Show list of all available EC2 commands:
+
+`aws ec2 help`
+
+- Show help for specific EC2 subcommand:
+
+`aws ec2 {{subcommand}} help`
+
+- Describe (print info about) a specified instance:
+
+`aws ec2 describe-instances --instance-ids {{instance_id}}`
+
+- Describe all instances:
+
+`aws ec2 describe-instances`
+
+- Describe all EC2 volumes:
```

Comment:
```suggestion
- Display all EC2 volumes:
```

Reply:
Changed very similar to how you suggested. Thanks for the feedback.